src/utils.php

```diff
@@ -154,3 +155,25 @@ function validateEditCarDataFields(string $name, int $year_made = 0, float $zero
     }
     return $err_msg;
 }
+
+function filterBrand(string $brand): string | false
+{
+    $model = new CarModel(make_db());
+    $cars = "";
+    if (empty($brand)) {
+        $cars = $model->getAllCarInfo();
+    } elseif (
+        $model->validateCarBrand($brand) ||
+        is_string($brand) ||
+        strlen($brand) <= 20 ||
+        strlen($brand) > 0
+    ) {
+        $cars = $model->filterCarBrand($brand);
+    }
+
+    if (!empty($cars)) {
+        return create_list_of_cars($cars);
+    } else {
+        return false;
+    }
```

Comment:
You can use an early return here. Just remove the else and return false after the if